### PR TITLE
Datachannels follow-up fixes

### PR DESCRIPTION
**Pull request by ferjm**
With this fixes we get closer to be spec compliant

### CODE REVIEW

**Comment by ferjm** [comment]
Ugh... this requires upgrading the gstreamer binaries for Travis

File: backends/gstreamer/datachannel.rs [hunk 1]
```diff
@@ -53,11 +55,30 @@ impl GStreamerWebRtcDataChannel {
...
+                (max_packet_life_time as u32).to_send_value(),
+            );
+        }
+
+        if let Some(max_retransmits) = init.max_retransmits {
+            init_struct.set_value("max-retransmits", (max_retransmits as u32).to_send_value());
+        }
+
+        if let Some(channel_id) = init.id {
+            init_struct.set_value("id", (channel_id as u32).to_send_value());
```
[the remark this answers is not in the document]
Comment (ferjm) [reply backends/gstreamer/datachannel.rs:77]: This is actually the [GstWebRTCDataChannel.id](https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/blob/master/ext/webrtc/gstwebrtcbin.c#L6450) param that matches the [RTCDataChannel.id](https://www.w3.org/TR/webrtc/#dom-rtcdatachannel-id) spec one. So I think we shouldn't change the naming here. I can change the one for the internal data channel id though.

**Comment by ferjm** [comment]
Given that we don't support Android in Servo anymore, I have disabled the Android build in Travis.

File: backends/gstreamer/datachannel.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Manishearth) [reply backends/gstreamer/datachannel.rs]: oh!  I see, my bad

**Comment by Manishearth** [comment]
r=me whenever ready

**Pull request merged**